src/components/dashboard/Dashboard.jsx

```diff
@@ -521,6 +524,30 @@ const Dashboard = props => {
     }
   }, [isCitizen])
 
+  useEffect(async () => {
+    const dontShowAgain = await AsyncStorage.getItem('dontShowWelcomeOffer')
+    if (!dontShowAgain) {
+      return
+    }
+
+    const country = await fetch('https://get.geojs.io/v1/ip/country.json')
+      .then(_ => _.json())
+      .then(_ => _.country)
+
+    const isEligible = supportedCountries.split(',').includes(country)
+
+    if (isWeb && welcomeOfferActive && isEligible) {
```

Comment:
why only web?

Reply:
Those are the eligibility requirements @decentralauren shared

Reply:
then make part of the feature flag config

Reply:
Done